src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  function clear() {
+    for (const key in state) {
+      delete state[key];
+    }
+  }
+
+  function removeSomeKeys(keysArray) {
```

Comment:
also it is considered a bad practice to mention dataType in the name of the variable - as it gives us very little context.
Here are some better options:

```suggestion
  function removeSomeKeys(keys) {
```

```suggestion
  function removeSomeKeys(keysToRemove) {
```